k8s/prod/common/pod-identity/pod-identity.yaml

```diff
@@ -0,0 +1,205 @@
+apiVersion: v1
```

Comment:
This is in here,  https://github.com/hmcts/cnp-flux-config/blob/4bcff1106fd637dc0cdfa1b60f801bb40e799c88/k8s/common/pod-identity/pod-identity.yaml